openupgrade_scripts/scripts/l10n_it/15.0.0.3/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,40 @@
+---Models in module 'l10n_it'---
+---Fields in module 'l10n_it'---
+# NOTHING TO DO
+l10n_it      / account.tax.report.line  / carry_over_condition_method (False): NEW selection_keys: ['always_carry_over_and_set_to_0', 'no_negative_amount_carry_over_condition', 'vp14_credit_carryover_condition', 'vp14_debt_carryover_condition'], mode: modify
+---XML records in module 'l10n_it'---
+# NOTHING TO DO
+NEW account.account.tag: l10n_it.account_tag_A_ATT
+NEW account.account.tag: l10n_it.account_tag_A_PASS
+NEW account.account.tag: l10n_it.account_tag_A_PL
+NEW account.account.tag: l10n_it.account_tag_BENI
+NEW account.account.tag: l10n_it.account_tag_B_ATT
+NEW account.account.tag: l10n_it.account_tag_B_PASS
+NEW account.account.tag: l10n_it.account_tag_B_PL
+NEW account.account.tag: l10n_it.account_tag_C_ATT
+NEW account.account.tag: l10n_it.account_tag_C_PASS
+NEW account.account.tag: l10n_it.account_tag_C_PL
+NEW account.account.tag: l10n_it.account_tag_D_ATT
+NEW account.account.tag: l10n_it.account_tag_D_PASS
+NEW account.account.tag: l10n_it.account_tag_D_PL
+NEW account.account.tag: l10n_it.account_tag_E_PASS
+NEW account.account.tag: l10n_it.account_tag_E_PL
+NEW account.account.tag: l10n_it.account_tag_IMPEGNI
+NEW account.account.tag: l10n_it.account_tag_RISCHI
+# DONE Deleted safely
```

Comment:
Put here `# nothing to do`, as ti references the NEW ones. The `# DONE Deleted safely` should go below the DEL ones.

Reply:
@MiquelRForgeFlow this is for not deleting them if they are used. That's why the method is called `safely`.

Reply:
> Still, the message "done" should go below the DEL lines.

Fixed in https://github.com/OCA/OpenUpgrade/pull/5098/commits/da3816994ae2f16bcfdaebfdfda510bf1e022ff8